src/converter.bat

```diff
@@ -0,0 +1,4 @@
+wsl make
+pause
+avrdude -c usbtiny -p m328p -U flash:w:miniboot.hex
```

Comment:
the programmer is avrdude in your case, some people may use something else. And the programmer as well. These are all variables you need to incorporate into the original Makefile.